pom.xml

```diff
@@ -51,7 +51,7 @@
              the built feature pack. -->
         <version.io.smallrye.graphql>2.9.1</version.io.smallrye.graphql>
         <version.org.eclipse.microprofile.graphql>2.0</version.org.eclipse.microprofile.graphql>
-        <version.com.graphql-java>22.1</version.com.graphql-java>
+        <version.com.graphql-java>22.2</version.com.graphql-java>
```

Comment:
Let's remove this for now please, we'll do this along with upgrading to sr-gql 2.10.0 next week. Otherwise, all looks good to me!